resource_planning/views/resource_planning_views.xml

```diff
@@ -154,6 +154,11 @@
           <field name="location" string="location" groups="resource_planning.group_multi_location"/>
           <filter string="Option" name="option" domain="[('state','=','option')]"/>
           <filter string="Booked" name="booked" domain="[('state','=','booked')]"/>
+          <filter string="Maintenance" name="maintenance" domain="[('state','=','maintenance')]"/>
+          <separator/>
+          <filter string="Currently" name="currently"
+            domain="[('date_start','&lt;=', datetime.datetime.combine(context_today(), datetime.time(23, 59, 59))),

```

Comment:
This does not do what the commit message says and I don't think it is correct.

If the resource is booked from today 10:00 to today 18:00, it won't show up in the _currently_ filter.

Reply:
So let suppose resource booked as you said : `date_start = today 10:00` and `date_end = today 18:00`.

The domain says : `date_start <= today 23:59:59 AND date_end >= today 00:00:00`, so the resource will show up.

Reply:
Yup, my bad. :-)